tests/swap_test.go

```diff
@@ -231,10 +164,10 @@ var _ = Describe("[Serial][sig-compute]SwapTest", Serial, decorators.SigCompute,
 			vmMemoryRequestkib := 512000
 			availableMemSizeKib := getAvailableMemSizeInKib(*targetNode)
 			availableSwapSizeKib := getSwapFreeSizeInKib(*targetNode)
-			swapSizeToUsekib := min(maxSwapSizeToUseKib, int(swapPartToUse*float64(availableSwapSizeKib)))
+			swapSizeToUsekib := math.Min(maxSwapSizeToUseKib, swapPartToUse*float64(availableSwapSizeKib))
```

Comment:
```suggestion
			swapSizeToUsekib := int(math.Min(maxSwapSizeToUseKib, swapPartToUse*float64(availableSwapSizeKib)))
```

so that all similarly-named variables have the same type.

Reply:
note please that since go 1.21 there is built-in `min` function (and we use go 1.21)
with `min` there was no need to change the current existing casts fwiw

Reply:
@RamLavi Can you please address the above mentioned comments?

Reply:
ah sorry, missed this one. On it

Reply:
@oshoval not sure why but the built in `min` function does not compile.

Reply:
maybe because of the updated casts, when i tried without touching the casts it did